src/Domain/BeginnerInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace PhpTuf\ComposerStager\Domain;
+
+interface BeginnerInterface
```

Comment:
I understand not documenting concrete implementations, but interfaces should probably be heavily documented. Can this interface and every method in it get doc comments?

Reply:
You're right. Let's do that.